src/openrct2/windows/TopToolbar.cpp

```diff
@@ -1890,6 +1890,20 @@ static void top_toolbar_tool_update_land_paint(sint16 x, sint16 y){
 *  rct2: 0x00664280
 */
 static void top_toolbar_tool_update_land(sint16 x, sint16 y){
```

Comment:
Braces go on their own lines.

Saying this once to avoid spam.

Reply:
I think with braces I followed the style of the code to avoid additional mess  that some (ol, untouched) functions have braces in the same line and new ones have in own line.... 

How should I proceed then?

Reply:
You update code you touch. You can leave code you didn't as-is.